gravitee-apim-console-webui/src/management/settings/groups/group/group.component.ajs.ts

```diff
@@ -355,12 +355,19 @@ const GroupComponentAjs: ng.IComponentOptions = {
             this.membersPageQuery.page = this.membersPage.page.current;
             this.members = this.membersPage.data;
             this.membershipState.sync(this.members);
+            this.isUserAdminAndMember = this.isCurrentUserGroupAdmin() && this.hasGroupMembership();
           })
           .finally(() => {
             this.membersLoaded = true;
           });
       };
 
+      this.isCurrentUserGroupAdmin = (): boolean =>
+        Array.isArray(this.members) && this.members.some((member) => member.id === this.currentUserId && member.roles.GROUP === 'ADMIN');
+
+      this.hasGroupMembership = (): boolean =>
+        Array.isArray(this.members) && this.members.some((member) => member.id === this.currentUserId);
+
```

Comment:
I think this condition is useless since the previous method already check for it.